### ISSUE TITLE
background.editor gutter

### ISSUE THREAD

**Issue by JeroenMBooij** [shalldie/vscode-background#513]
I installed Version 2.0.3 and when I use the background.editor the image doesn't cover the line gutter.

![Screenshot 2024-12-28 at 16 52 40](https://github.com/user-attachments/assets/5217d2d9-acb0-48a1-a5c1-13bbc91365b0)

![Screenshot 2024-12-28 at 16 52 29](https://github.com/user-attachments/assets/aa379a80-1eb9-46a6-a4bf-a23358e78fcc)

Using the "position: fixed" does allow me to put the image over the gutter, but it does not allow me to use the  "background-size: cover" property, so the image doesn't fit the editor window

![Screenshot 2024-12-28 at 16 54 24](https://github.com/user-attachments/assets/4ff47d45-766e-4718-aad3-02ba998481e8)
![Screenshot 2024-12-28 at 16 53 48](https://github.com/user-attachments/assets/77282f5c-6fe8-438f-a2e3-5838e90d4eee)

**Comment by shalldie** [comment]
Yes `"position": "fixed"` has effected `width: 100%`/`height:100%`, which effects `"background-size": "cover"`.

It is not recommended to change the position as it has a significant impact on positioning.
Because it is necessary to focus on the masking effect of the code part, the editor part abandoned the line number part in the design.
At present, the editor's images cannot affect it.
